client/device/camera/camera.js

```diff
@@ -0,0 +1,23 @@
+(function (window) {
+  /**
+   * Set correct image orientation.
+   */
+  var resetOrientation = function (imgSrc, orientation, callback) {
+  };
+
+  /**
+   * Get orientation value from captured image.
+   */
+  var getOrientation = function (file, callback) {
+  };
+
+  /**
+   * API for control user mobile camera device.
+   */
+  var camera = {
+    setImage : function (inputId, callback) {
```

Comment:
As we discussed, this function's signature should be func(parentElement, callback).

But please do it in the follow-up CL.